### PR TITLE
Release 1 0 4

### PR DESCRIPTION
**Pull request by aureate-labs-team**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SHOPIFY_DOMAIN= 'vsf-next-pwa.myshopify.com',
```
Comment (sevensilvers) [inline .env:1]: We shouldn't push this one for template

File: .nuxt/App.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
+import Vue from 'vue'
```
Comment (sevensilvers) [inline .nuxt/App.js]: .nuxt directory shouldn't be push to template too.

**Review by sevensilvers — CHANGES_REQUESTED** [review]
Most of the changes here for 1 0 4 are already from the main from @Mateuszpietrusinski last chore update with the terms and conditions page. I suggest we could drop this unless you guys added more commit in here. Thanks

File: .env (hunk 1 above)
Comment (aureate-labs-team) [reply .env:1]: We've added because its having dependency in nuxt.config.js

File: .nuxt/App.js (hunk 2 above)
Comment (aureate-labs-team) [reply .nuxt/App.js]: Yap, its removed

File: integrations/refresh-caching.client.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export default async({ $cookies, $config }) => {
```
[the remark this answers is not in the document]
Comment (aureate-labs-team) [reply integrations/refresh-caching.client.js:1]: We've faced problems with SW caching to reflect new changes on each production deployment. So we've reset cache if the same build version is not there on the local browser.

**Pull request merged**